src/app/tasks/task-submission-viewer/task-submission-viewer.component.html

```diff
@@ -1,29 +1,27 @@
 <div class="panel-body task-submission-viewer">
-  <div class="large-notice-block" ng-show="notSubmitted(task) || task.processing_pdf || loadingDetails(task)">
-    <div ng-show="notSubmitted(task) && ! loadingDetails(task)">
+  <div class="large-notice-block" [hidden]="!(notSubmitted(task) || task.processing_pdf || task.needsSubmissionDetails())">
+    <div [hidden]="!(notSubmitted(task) && ! task.needsSubmissionDetails())">
       <i class="fa fa-upload fa-2x"></i>
       <p>Upload submission before viewing</p>
     </div><!--/no-pdf-->
-    <div ng-show="task.processing_pdf && ! loadingDetails(task)">
+    <div [hidden]="!(task.processing_pdf && ! task.needsSubmissionDetails())">
         <i class="fa fa-clock-o fa-2x"></i>
         <p>
           Submission is still being processed
-          <p class="small text-muted">
-            Please check again shortly
-          </p>
+          <p class = "muted-text mat-small">Please check again shortly
```

Comment:
These are still bootstrap styles that need to be migrated to Angular Material Typography